src/System.Private.CoreLib/src/System/Utf8String.Construction.cs

```diff
@@ -0,0 +1,241 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Text.Unicode;
+
+namespace System
+{
+    public sealed partial class Utf8String
+    {
+        /*
+         * CONSTRUCTORS
```

Comment:
Assuming that this will move the shared partition eventually, this is not the best place to document this since it applies to a single runtime only. (The plumbing for Mono or CoreRT will be different.)

Reply:
That's fair. Would it be ok to punt this until we do move it into the shared partition and have a better view of what ctor support might look like on those other platforms?

Reply:
I would move this comment to somewhere in VM, e.g. ecall.cpp around PopulateManagedStringConstructors

Reply:
It applies equally to both String and Utf8String.